package.json

```diff
@@ -20,6 +20,8 @@
     "dependencies": {
         "bluebird": "^3.5.1",
         "cheerio": "^0.22.0",
+        "eslint-scope": "^4.0.0",
```

Comment:
These two dependencies seem to have been added for `eslint`, which is listed as a development dependency. You need to put all eslint dependencies in the same section, either here if we're going to use `eslint` in production or, in the devDependencies section.